features/step_definitions/bookings_steps.rb

```diff
@@ -0,0 +1,32 @@
+Given('I am a registered user') do
+  @user = User.create(first_name: 'user', last_name: 'one', email: 'user1@test.com')
+end
+
+Given('A meeting room is available') do
+  @meeting_room = MeetingRoom.create(name: 'MR1', capacity: 10, is_available: true)
+end
+
+Given('A meeting room is unavailable') do
```

Comment:
Try combining these two step definitions.

Reply:
Yes will do, even I thought of same today.